packages/@lwc/wds-core/src/browser/wireMockUtil.js

```diff
@@ -0,0 +1,185 @@
+export function getWireMockUtil() {
```

Comment:
My feedback before may not have been specific enough. Could you move everything from `let initialValue = undefined;` and below into its own file? Then we can read it from disk using `fs.readFileSync` or something like that. The benefit here, is it can be linted as JavaScript, and can even be tested independently in the future.